x/dispensation/client/cli/query.go

```diff
@@ -89,7 +89,7 @@ func GetCmdDistributionRecordForRecipient(queryRoute string) *cobra.Command {
 func GetCmdDistributionRecordForDistName(queryRoute string) *cobra.Command {
 	return &cobra.Command{
 		Use:   "records-by-name [distribution name] [status]",
-		Short: "get a list of all distribution records .Status : [Completed/Pending/All]",
+		Short: "get a list of all distribution records Status : [Completed/Pending/All]",
```

Comment:
"get a list of all distribution records *by* status (completed/pending/all)" <- does the caller pass in these words for status? If not needs a numbers -> status mapping in the description.

Reply:
They do pass in the words , which would be more intuitive to use compared to numbers 
```
func GetDistributionStatus(status string) DistributionStatus {
	switch status {
	case "Completed":
		return DistributionStatus_DISTRIBUTION_STATUS_COMPLETED
	case "Pending":
		return DistributionStatus_DISTRIBUTION_STATUS_PENDING
	case "Failed":
		return DistributionStatus_DISTRIBUTION_STATUS_FAILED
	default:
		return DistributionStatus_DISTRIBUTION_STATUS_UNSPECIFIED
	}
}
```